chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -494,23 +494,49 @@ mcCombinations.prototype.possibleCombinations = [
 		maxfields: 1,
 		score: 0,
 		autoSubmit: false
+	},
+	{
+		combinationId: 'form001',
```

Comment:
any reason you added this? this represents quite a major change to our detection algo.